backend/drive/drive.go

```diff
@@ -684,8 +691,16 @@ func (f *Fs) list(ctx context.Context, dirIDs []string, title string, directorie
 		if parentsQuery.Len() > 1 {
 			_, _ = parentsQuery.WriteString(" or ")
 		}
-		if f.opt.SharedWithMe && dirID == f.rootFolderID {
-			_, _ = parentsQuery.WriteString("sharedWithMe=true")
+		if dirID == f.rootFolderID {
```

Comment:
I don't think the logic is correct here...

if `dirID == f.rootFolderID` and `!f.opt.SharedWithMe` then it should be outputing `fmt.Fprintf(parentsQuery, "'%s' in parents", dirID)` but in this case it outputs nothing.

I think the condition needs to be

`if (f.opt.SharedWithMe || f.opt.StarredOnly) && dirID == f.rootFolderID {`

Reply:
Yes, perfect.  I was going to talk about this actually, but you beat me to it.  This logic works for me.